azure_functions_worker/constants.py

```diff
@@ -1,3 +1,10 @@
 # Capabilities
 RAW_HTTP_BODY_BYTES = "RawHttpBodyBytes"
 TYPED_DATA_COLLECTION = "TypedDataCollection"
+
+# User packages
+PKGS_PATH = "site/wwwroot/.python_packages"
+VENV_PKGS_PATH = "site/wwwroot/worker_venv"
+
+PKGS_36 = "lib/python3.6/site-packages"
+PKGS_37 = "lib/python3.7/site-packages"
```

Comment:
In PGKS_PATH, we want to drop the version specific package path.
We can add a python path to support **lib/site-packages** as well, so that we will have a transition windows for updating our toolings.